ci-operator/step-registry/openstack-k8s-operators/gather/openstack-k8s-operators-gather-commands.sh

```diff
@@ -78,12 +78,24 @@ pushd "${BASE_DIR}" || exit
 mkdir logs
 pushd logs || exit
 
+# check if we have any kuttl namespace
+KUTTL_NS=$(oc get project -o name | grep kuttl)
```

Comment:
My output looks like this (with grep openstack), so we might need to add a 'cut' here?
```
$ oc get project -o name | grep openstack
project.project.openshift.io/openshift-openstack-infra
project.project.openshift.io/openstack
project.project.openshift.io/openstack-operators
```

Reply:
good catch, thanks!